doc/version_history.rst

```diff
@@ -6,6 +6,17 @@
 Version History
 ###############
 
+v0.1.2
+======
+
+* Add support for start, disable, enable and standby commands.
+
+Requires:
+
+* ts_salobj
+* ts_tcpip
```

Comment:
I think it might be good to have the version number for `ts_tcpip` because the `ts_attcpip` depends on it closely.

Reply:
You're right. I added it again.